README.md

```diff
@@ -114,6 +114,15 @@ ReactiveCocoa to your `Podfile`:
 pod 'ReactiveCocoa', '~> 9.0'
 ```
 
+#### Swift Package Manager
+
+If you use Swift Package Manager, simply add ReactiveCocoa as a dependency
+of your package in `Package.swift`:
+
+```
+.package(url: "https://github.com/ReactiveCocoa/ReactiveCocoa.git", branch: "master")
```

Comment:
we should update the version next, then we can update these instructions to point to the right version